doc/api/cli.md

```diff
@@ -2018,6 +2018,31 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector in MiB.
+Increasing the semi-space size by 1MiB will
+cause the heap size of V8 to increase by 3MiB,
```

Comment:
Formatted size (with trailing space before unit) for consistency with `cli.md`.

---
```suggestion
cause the heap size of V8 to increase by 3 MiB,
```